### DISCUSSION TITLE
OpenRadioss vs Altair Radioss Runtime

### DISCUSSION THREAD

**Discussion by bocaj2222** [OpenRadioss/OpenRadioss#794] [Q&A]
Hello,

The Windows release of OpenRadioss is very exciting, thank you. I've done some comparisons to Atlair Radioss. It appears OpenRadioss runs 30-50% slower when using Intel MPI. I tried to match environment variables and settings to the Altair runs, including using the same MPI installation, but did not see any improvement. Run times between Altair Radioss and OpenRadioss appear nearly identical when using SMP. I'm curious if this is an inherent difference between the two MPI versions of Radioss, or if there could be something I'm missing. 

As a specific example, the cell phone drop test example takes about 100s to run with OpenRadioss and 70s with Altair Radioss, both with 32 cores (-np 32). Thanks.

**Comment by elequiniou** [comment]
Dear bocaj2222, with bigger models longer to run, this is not our findings, the gap should be smaller, the team will investigate this example. Thanks a lot for your interest in OpenRadioss and for your appreciated feedback!
Eric.

**Comment by OlivierWienholtz** [comment]
What IntelMPI version did you install ? 
Did you use a Released binary or your own built library ?

**Reply by bocaj2222** [reply]
I am using the first released windows binary (9aaa721). The intel oneapi version is 2022.3.1. However, I seem to get similar results when using that version of intelMPI, or the one installed alongside Altair Radioss 2022. I reran the following scenarios this morning with cell phone drop test (ran a little better than yesterday):

OpenRadioss Intel MPI 2022.3.1: 85.8s
OpenRadioss Intel MPI (installed alongside Altair Radioss 2022): 83.9s
Altair Radioss Intel MPI (installed alongside Altair Radioss 2022): 64.7s
Altair Radioss Intel MPI 2022.3.1: 70.9s

With one of my larger models, the OpenRadioss took 24% longer to run than Altair Radioss.

**Comment by OlivierWienholtz** [comment]
Can you pls check that you have a Minus value to /PRINT in *1.rad file.
This enables Elapse Time/Remaining time in Stdout.
Save the logs + 1.out files & send them here... D'like to check how timers, behave.

**Reply by bocaj2222** [reply]
[Cell_Phone_Drop_Comparison.zip](https://github.com/OpenRadioss/OpenRadioss/files/10044876/Cell_Phone_Drop_Comparison.zip)

Sure, see attached. I appreciate the help.

**Comment by DavidR-Vortex** [comment]
Are you running the same level of precision for both?

**Reply by bocaj2222** [reply]
I've used default settings for both, which should be double precision. Are you able to test and share results with the cell phone drop test as well?
